indico/modules/events/registration/models/form_fields.py

```diff
@@ -105,6 +105,10 @@ def view_data(self):
     def html_field_name(self):
         return 'field_{}'.format(self.id)
 
+    @property
+    def filter_choices(self):
```

Comment:
This is really ugly. Why not simply return `None` in the base class' `filter_choices` property? Then you can get rid of this property and simply use `whatever.field_impl.filter_choices`